src/module/transaction/screen/SendConfirmationScreen/SendConfirmationScreen.tsx

```diff
@@ -1,80 +1,55 @@
-import { Col, Typography, useModal } from "@peersyst/react-native-components";
+import { Col, Typography, useModal, useToast } from "@peersyst/react-native-components";
 import CountdownButton from "module/common/component/input/CountdownButton/CountdownButton";
 import { useRecoilValue } from "recoil";
 import sendState from "module/transaction/state/SendState";
-import useSendTransaction from "../../query/useSendTransaction";
-import SendModal from "module/transaction/component/core/SendModal/SendModal";
-import LoadingModal from "module/common/component/feedback/LoadingModal/LoadingModal";
 import useWalletState from "module/wallet/hook/useWalletState";
 import SendSummary from "./SendSummary";
-import settingsState from "module/settings/state/SettingsState";
-import ConfirmPinModal from "module/settings/components/core/ConfirmPinModal/ConfirmPinModal";
-import { useState } from "react";
 import { useTranslate } from "module/common/hook/useTranslate";
+import SendTransactionModal from "module/transaction/component/feedback/SendTransactionModal/SendTransactionModal";
+import SendModal from "module/transaction/component/core/SendModal/SendModal";
+import { useSendTransaction } from "module/transaction/hook/useSendTransaction";
 
 const SendConfirmationScreen = (): JSX.Element => {
-    const [showConfirmation, setShowConfirmation] = useState(false);
     const translate = useTranslate();
-    const [loading, setLoading] = useState(false);
-
-    const { amount, fee: fee, senderWalletIndex, receiverAddress, message, token } = useRecoilValue(sendState);
-    const { fee: feeInDecimals } = useRecoilValue(settingsState);
+    const { hideModal } = useModal();
+    const { showToast } = useToast();
+    const sendStateValue = useRecoilValue(sendState);
+    const { sendTransaction, ...rest } = useSendTransaction(sendStateValue);
     const {
         state: { wallets },
     } = useWalletState();
+
+    const { asset, amount, receiverAddress, senderWalletIndex } = sendStateValue;
     const senderWallet = wallets[senderWalletIndex!];
     const { account: senderName } = senderWallet;
-    const { mutate: sendTransaction, isLoading, isSuccess, isError } = useSendTransaction(senderWalletIndex!);
-    const { hideModal } = useModal();
 
-    const handleConfirmation = async () => {
-        sendTransaction(
-            {
-                amount: amount!, //TODO: revise the unit (yocto vs near)
-                message: message!,
-                to: receiverAddress!,
-                feeRate: feeInDecimals,
-            },
-            {
-                onSettled: () => setLoading(false),
-            },
-        );
-    };
+    function closeModal() {
+        hideModal(SendModal.id);
```

Comment:
Ojo el copy and paste del `SendModal`. Lo has de adaptar al stake. Tmb debería ir a la screen de success no cerrar el modal.